pyseir/cli.py

```diff
@@ -60,24 +61,42 @@ def _generate_state_reports(state=None):
         report.generate_report()
     else:
         for state in us.states.STATES:
-            _generate_state_reports(state.title())
+            _generate_state_reports(state.name.title())
```

Comment:
Should this be state.name.title or state.title? Seems you are using state.title somewhere else.

Reply:
fixed.